R/fromRes.R

```diff
@@ -3,29 +3,42 @@
 
 ### called by getRestuls() and plotICC() ---------------------------------------
 
-eapFromRes <- function ( resultsObj, idVarName = NULL, verbose = TRUE ) {
-  eapRo<- setdiff(intersect( which(resultsObj[,"par"] == "eap"),which(resultsObj[,"indicator.group"] == "persons")), which(resultsObj[,"derived.par"] == "rel"))
-  id   <- unique(resultsObj[intersect(which(resultsObj[,"type"] == "tech"), which(resultsObj[,"par"] == "ID")),"derived.par"])
-  id   <- getIdVarName(id, idVarName, verbose=verbose)
-  if ( length ( eapRo ) == 0 ) {
+eapFromRes <- function (resultsObj, idVarName = NULL, verbose = TRUE){
+  checkmate::assert_character(idVarName, null.ok = TRUE)
```

Comment:
I think adding `len = 1` is useful here, i.e. `checkmate::assert_character(idVarName, null.ok = TRUE, len = 1)`.

Reply:
should I do that for every `idVarName` object for all functions?

Reply:
yes, for all user-level functions that have `idVarName` as argument. Thanks!